web/models/clusters_settings.go

```diff
@@ -0,0 +1,15 @@
+package models
+
+type ClusterSettings struct {
+	ID             string                     `json:"id"`
+	SelectedChecks []string                   `json:"selected_checks"`
+	Hosts          []*ConnectionInfoAwareHost `json:"hosts"`
+}
+
+type ConnectionInfoAwareHost struct {
+	Name    string `json:"name"`
+	Address string `json:"address"`
+	User    string `json:"user"`
+}
```

Comment:
I think `ConnectionInfoAwareHost` is really hard to understand.
Hmm, maybe `ClusterSettings` just has `[]*HostConnection`. 

Then `HostConnection` will likely have additional fields for SSH keys in the future.

Reply:
I agree with @stefanotorresi `HostConnection` sounds good!

Reply:
`HostConnection` SOLD!